src/arrays.js

```diff
@@ -1,61 +1,88 @@
 const getNthElement = (index, array) => {
   // your code here
+  const newIndex = index % array.length;
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
   // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
   // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
   // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
   // your code here
+  const newArray = [...array];
+  newArray.push(element);
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
   // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
   // your code here
+  return numbers.map(x => String(x));
 };
 
 const uppercaseWordsInArray = strings => {
   // your code here
+  return strings.map(x => x.toUpperCase());
```

Comment:
As above, should use a clearer variable name here

Rather than saying the same thing everywhere else in the repo, anywhere youre using something abstract like `x` for a variable like this should be updated